docs/getting_started.md

```diff
@@ -141,14 +148,16 @@ You can check the status of your configuration with:
 ```
 
 ### Verifying the AWS Adapter installation
-Perform the below steps to see if the adapter is installed correctly
+
+Perform the below steps to see if the adapter is installed correctly.
+
 ```bash
-# check if the controller pod is running
+# check if the controller pod is running.
 kubectl get pods -n nirmata-aws-adapter
 NAME                                              READY   STATUS    RESTARTS   AGE
 aws-adapter-kyverno-aws-adapter-997f45bb9-c2z5j   1/1     Running   0          109m
```

Comment:
Can you change the pod name to whatever is the latest convention?

Reply:
Done.